schema/image/resized.js

```diff
@@ -17,7 +17,7 @@ export const resizedImageUrl = (image, options) => {
   const desired = _.pick(opts, 'width', 'height');
   const factor = _.min(_.map(desired, (value, attr) => {
     return value / image[`original_${attr}`];
-  }));
+  })) || Infinity;
```

Comment:
Hm; yeah I'm not sure but the specs cover this object pretty well so if we pass there it should be fine.